src/inmanta/server/services/environment_metrics_service.py

```diff
@@ -166,7 +155,11 @@ def create_metrics_gauge(metric_values_gauge: Sequence[MetricValue], timestamp:
             for mv in metric_values_gauge:
                 metric_gauge.append(
                     EnvironmentMetricsGauge(
-                        metric_name=mv.metric_name, timestamp=timestamp, count=mv.count, environment=mv.environment
+                        metric_name=mv.metric_name,
+                        grouped_by=mv.grouped_by if mv.grouped_by else "None",
```

Comment:
Let's turn this special value into constant. We can then give it a clear name and add a comment to it to make it clear why we have it.

```suggestion
                        grouped_by=mv.grouped_by if mv.grouped_by else "None",
```